### ISSUE TITLE
`ServiceCardImage`, `UserAvatar`, and `EntityAvatar` should forward ref to `Image` component

### ISSUE THREAD

**Issue by tomgenoni** [thumbtack/thumbprint#324]
Needed to support using `ImageServicePicture`:

```
<ImageServicePicture id="123">
    {({ src, sources, ref }) => (
        <ServiceCardImage
            src={src}
            sources={sources}
            alt={item.text}
            ref={ref}
        />
    )}
</ImageServicePicture>

**Comment by danoc** [comment]
Doing this to Avatar is blocked by #307.

**Comment by danoc** [comment]
This can now be done in `Avatar,` too.

**Comment by danoc** [comment]
Fixed in #334.
